config/import-control.xml

```diff
@@ -170,13 +170,20 @@
     <allow pkg="javax.swing"/>
     <allow pkg="java.util" exact-match="true"/>
     <allow pkg="com.puppycrawl.tools.checkstyle.api" local-only="true"/>
-    <allow pkg="com.puppycrawl.tools.checkstyle.utils" local-only="true"/>
-    <allow class="com.puppycrawl.tools.checkstyle.JavaParser" local-only="true"/>
     <allow class="com.puppycrawl.tools.checkstyle.gui.MainFrameModel.ParseMode"
            local-only="true"/>
-    <allow class="com.puppycrawl.tools.checkstyle.JavadocDetailNodeParser" local-only="true"/>
     <allow class="com.google.common.collect.ImmutableList" local-only="true"/>
-    <allow class="java.nio.charset.StandardCharsets" local-only="true" />
+    <disallow pkg="com\.puppycrawl\.tools\.checkstyle\.(checks|ant|doclets|filters)" regex="true"/>
+
+    <file name=".*(Model|Presentation)" regex="true">
+      <disallow pkg="java.awt.Graphics"/>
+      <disallow pkg="java.awt.Component"/>
+      <disallow pkg="javax\.swing\.J.*" regex="true"/>
+      <allow class="java.nio.charset.StandardCharsets" />
+      <allow class="com.puppycrawl.tools.checkstyle.JavadocDetailNodeParser"/>
+      <allow class="com.puppycrawl.tools.checkstyle.JavaParser"/>
+      <allow pkg="com.puppycrawl.tools.checkstyle.utils"/>
+    </file>
```

Comment:
please show power of your functionality to enforce limit of Closeable/Flushable to only for this 
```
$ grep -rl "Closeable" * --exclude-dir={target,resources,xdocs,resources-noncompilable,config,test}
src/main/java/com/puppycrawl/tools/checkstyle/PropertyCacheFile.java
src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

$ grep -rl "Flushable" * --exclude-dir={target,resources,xdocs,resources-noncompilable,config,test}
src/main/java/com/puppycrawl/tools/checkstyle/PropertyCacheFile.java
```

Reply:
> src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

This is `java.io.Closeable` not `com.google.common.io.Closeables`.

`PropertyCacheFile` was done.